app/models/concerns/edm/entity/depiction.rb

```diff
@@ -17,7 +17,7 @@ def thumbnail_src
       end
 
       def thumbnail_full
-        api_response[:depiction][:id]
+        api_response.key?(:depiction) ? api_response[:depiction][:id] : nil
```

Comment:
for this we could use:
`api_response.dig(:depiction, :id)`

https://ruby-doc.org/core-2.3.0/Hash.html#method-i-dig